stories/examples/controlled.js

```diff
@@ -0,0 +1,199 @@
+import React, {Component} from 'react'
```

Comment:
Pull this down and run it. It's actually a pretty slick experience.

```
git clone https://github.com/paypal/downshift.git
cd downshift
npm install
npm run storybook
```

Then open `http://localhost:6006` and choose `Examples => controlled` :+1: